### DISCUSSION TITLE
Potential approach to crediting an artist who makes badges for a whole set

### DISCUSSION THREAD

**Discussion by cIymax** [RetroAchievements/RAWeb#1588] [Ideas]
**Background**

> Searo — 12/16/2021 8:04 AM
I also agree that credit should be given on the game page and/or achievement pages. Even the forum page would be a nice minimum  acknowledgement that could probably be standardized when things are updated or art requests are fulfilled for a dev.
>
> It's just due to how everything currently works  being tied to achievements and not separate at all it would be a lot of work plus there would be no automation in our current state since not all people working on badges can actually upload them which means every achievement/icon would need the uploader to manually update who the badge/icon author is for each one. Maybe someone has a nice solution but I feel like this is one that will likely be waiting until V2.

[source](https://discord.com/channels/310192285306454017/748750711566893066/921024915350642699) via Discord

**Ideas**
- Allow a collaboration claim to be made by an artist on a set that a dev already has a primary claim on (once they've agreed to work together).  The artist should have an appropriate art-helper role on the website.
- When the set is published, all cheevos will give artist credit to the artist.
- I'm not sure how common the use case is for 1 dev to collab with 1 artist, but a solution can encourage more such collabs in the future.
- I'm bringing this up because I have a set revision/addition in mind that could use the help of an artist.  It would be great if they could be credited on the page, and it'd also increase the chances that devs can get the artistic support they need.
- If this approach is pursued, we'd probably want to characterize a claim type.  That way, a dev can be a dev collaborator for one game but an artistic collaborator for another game.
- I understand that this won't address icon gauntlets or devs drawing some of their badges but asking for help on certain badges.  But at least we'll have a solution in place for new sets or new achievements being added to an existing set.

[Link](https://discord.com/channels/310192285306454017/1106523610006437975) to companion topic on Discord 

See also #1096 and #689.  I made this as a separate topic, because #1096 talks about credit being manually entered by a dev, while #689 talks about having a game page credit various types of contributors but just in a general sense.

**Sample**

On the game page, the Artist field is only displayed if the Author is not also the Artist.
 
![image](https://github.com/RetroAchievements/RAWeb/assets/94894395/a8f26e3e-309a-45de-bba4-5eddd2409202)

**Comment by wescopeland** [comment]
I don't think this fully solves the problem, and it perhaps creates new ones.

A great number of sets have multiple artists. Now the credit for all the game's art is mixed.
